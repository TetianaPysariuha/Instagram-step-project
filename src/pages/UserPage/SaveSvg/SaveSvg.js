function SaveSvg() {
  return <svg aria-label="" className="_ab6-" color="#262626" fill="#262626" height="12" role="img" viewBox="0 0 24 24" width="12"><polygon fill="none" points="20 21 12 13.44 4 21 4 3 20 3 20 21" stroke="currentColor" strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" /></svg>;
}

export default SaveSvg;
