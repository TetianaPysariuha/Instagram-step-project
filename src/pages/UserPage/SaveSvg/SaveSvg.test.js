import { render } from '@testing-library/react';
import SaveSvg from './SaveSvg';

describe('SaveSvg renders', () => {
  test('Should SaveSvg render', () => {
    const { asFragment } = render(<SaveSvg />);
    expect(asFragment()).toMatchSnapshot();
  });
});
