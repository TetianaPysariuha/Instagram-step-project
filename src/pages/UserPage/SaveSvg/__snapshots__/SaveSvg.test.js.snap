// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SaveSvg renders Should SaveSvg render 1`] = `
<DocumentFragment>
  <svg
    aria-label=""
    class="_ab6-"
    color="#262626"
    fill="#262626"
    height="12"
    role="img"
    viewBox="0 0 24 24"
    width="12"
  >
    <polygon
      fill="none"
      points="20 21 12 13.44 4 21 4 3 20 3 20 21"
      stroke="currentColor"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2"
    />
  </svg>
</DocumentFragment>
`;
